Test function failure after commit

Test function failed with the following details:
`=================================== FAILURES ===================================
________________________________ test_subtract _________________________________
    def test_subtract():
>       assert subtract(2, 3) == -1
E       assert 5 == -1
E        +  where 5 = subtract(2, 3)
test_functions.py:12: AssertionError
---------- coverage: platform linux, python 3.10.10-final-0 ----------
Coverage XML written to file coverage.xml
=========================== short test summary info ============================
FAILED test_functions.py::test_subtract - assert 5 == -1
 +  where 5 = subtract(2, 3)
========================= 1 failed, 1 passed in 0.05s ==========================
Error: Process completed with exit code 1.`

--- a/functions.py
+++ b/functions.py
@@ -2,7 +2,7 @@ def add(a, b):
     return a + b
 
 def subtract(a, b):
-    return a + b  # <--- fix this in step 7
+    return a - b  # <--- fix this in step 7
 
 def multiply(a, b):
     return a * b

